src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -2932,27 +3105,49 @@ public void removeConnection(String uuid) {
     return ret;
   }
 
-  static String formatConnection(Map<String, Object> c) {
-    return String.format("%s %s %s", c.get("id"), c.get("uri"), c.get("uuid"));
+  /**
+   * list connections - current connection names to this mrl runtime
+   * 
+   * @return
+   */
+  public Map<String, List<Map<String, String>>> lc() {
+    return getConnectionHeaders();
   }
 
-  public String getConnectionName(String uuid) {
+  public Map<String, String> getConnectionHeader(String uuid) {

```

Comment:
would it ever be possible to have multiple headers with the same key? or are we forbidding that?

Reply:
absolutely not - a connection is identified by a uuid